org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/xsd/XSDValidationExtensionsTest.java

```diff
@@ -348,6 +348,28 @@ public void src_import_1_2_different_range() throws BadLocationException {
 		testCodeActionsFor(xml, d, ca(d, te(2, 11, 2, 11, " namespace=\"\"")), ca(d, te(1, 54, 1, 54, " targetNamespace=\"\"")));
 	}
 
+	@Test

```

Comment:
Also include a test for the `xs:include` element (it has a `schemaLocation` attribute just like `xs:import`, and this PR also fixes the error range for it).

Another thing that might be good to test is what happens when `schemaLocation` is a blank String (`""`).

Reply:
Ignore the comment about blank schemaLocation; that generates a different error that should be handled in a seperate PR.